tools/yaml-templates/security.yml

```diff
@@ -8,7 +8,12 @@ steps:
       Configuration: 'required'
       TargetType: 'eslint'
       TargetsESLint: '$(Build.SourcesDirectory)/**/*.{js,jsx,ts,tsx}'
+      Parser: '@typescript-eslint/parser'
+      ParserOptions: |
+        sourceType:module
+        ecmaVersion:2018
```

Comment:
It seems elsewhere in the repo (`.eslintrc.json`, `GuardianCustomConfiguration.json` has ecmaVersion set to 12 (aka 2021), did we want to keep it consistent? 

Also we might want to decide what should go in `GuardianCustomConfiguration.json`, like if we want to just override them by specifying them here or if we want to specify them in the `GuardianCustomConfiguration.json` instead

Reply:
Where do we set the ecmaVersion to 12? The only place in the repo I can find the string "ecmaVersion" is the root .eslintrc.js file and it's set to 2018.

Reply:
Also I can't find a GuardianCustomConfiguration.json file anywhere in the repo. Am I overlooking some location? As of right now I'm not sure I can think of a reason why putting these settings in there would be better/worse than putting them in the YAML.

Reply:
I'm curious if @aeunms was searching a different repo. I find results for GuardianCustomConfiguration.json in the other repo that uses this ESLint task, but not in the microsoft-teams-library-js repo. I also can't find any other reference to `ecmaVersion` that doesn't use version 2018.